docs/model_configuration.md

```diff
@@ -89,6 +89,21 @@ directory containing the model.  The required values for *platform*
 and *backend* are described in the [backend
 documentation](https://github.com/triton-inference-server/backend/blob/main/README.md#backends).
 
+### Model Transaction Policy
+
+The *model_transaction_policy* property describes the nature of
+transactions expected from the model.
+
+#### Decoupled
+
```

Comment:
"This boolean setting indicates whether responses generated by
the model are [decoupled](docs/decoupled_backends_and_models.md)
with the requests issued to it. The number of
responses generated by the model may differ from number of requests
issued, and the responses may be out of order relative to
the order of requests."